podcasts/End of Year/Stories/Views/PaidStoryWallView.swift

```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+import PocketCastsServer
+
+struct PaidStoryWallView: View {
+    var body: some View {
+        GeometryReader { geometry in
+            PodcastCoverContainer(geometry: geometry) {
+                StoryLabelContainer(geometry: geometry) {
+                    SubscriptionBadge(tier: .plus, displayMode: .gradient, foregroundColor: .black)
+                    StoryLabel(L10n.eoyTheresMore, for: .title, geometry: geometry)
+                    StoryLabel(L10n.eoySubscribeToPlus, for: .subtitle, color: Color(hex: "8F97A4"), geometry: geometry)
+                }
+                .padding(.bottom, geometry.size.height * 0.06)
+
+                Button(L10n.eoyStartYourFreeTrial) {
+                    guard let storiesViewController = SceneHelper.rootViewController()?.presentedViewController else {
+                        return
+                    }
+
+                    NavigationManager.sharedManager.showUpsellView(from: storiesViewController, source: .endOfYear, flow: SyncManager.isUserLoggedIn() ? .endOfYearUpsell : .endOfYear)
+
+                }
+                .buttonStyle(StoriesButtonStyle(color: .black, icon: nil))
+            }
+        }
+        .background(
+            ZStack {
+                LinearGradient(
+                    stops: [
+                        Gradient.Stop(color: .black, location: 0.00),
+                        Gradient.Stop(color: .black.opacity(0), location: 1.00),
+                    ],
+                    startPoint: UnitPoint(x: 0.5, y: 0.24),
+                    endPoint: UnitPoint(x: 0.5, y: 1.04)
+                )
+
+                VisualEffectView(effect: UIBlurEffect(style: .systemThinMaterialDark))
+            }
+            .allowsHitTesting(false)
+            .ignoresSafeArea(edges: [.top, .bottom])
+        )
```

Comment:
I feel like we can just use the `ultraThinMaterial` as the background without the gradient which allows more of the background to show through but still ensures the text is legible:

| Before | After |
|:---:|:---:|
|<img src="https://github.com/Automattic/pocket-casts-ios/assets/793774/928a94ef-2e09-4b3e-ae08-4acac912466f" width="320" />|<img src="https://github.com/Automattic/pocket-casts-ios/assets/793774/8d8b1992-9163-4a45-903f-646466e35280" width="320" />|


```suggestion
        .background(.ultraThinMaterial)
        .environment(\.colorScheme, .dark)
```